web-regresjonstest/src/test/kotlin/no/nav/su/se/bakover/web/søknadsbehandling/OpprettNySakMedSøknadOgSøknadsbehandlingLokalt.kt

```diff
@@ -0,0 +1,117 @@
+package no.nav.su.se.bakover.web.søknadsbehandling
+
+import ch.qos.logback.classic.util.ContextInitializer
+import no.nav.su.se.bakover.common.endOfMonth
+import no.nav.su.se.bakover.common.startOfMonth
+import no.nav.su.se.bakover.domain.Fnr
+import no.nav.su.se.bakover.test.generer
+import no.nav.su.se.bakover.web.SharedRegressionTestData.withTestApplicationAndDockerDb
+import no.nav.su.se.bakover.web.søknad.ny.NySøknadJson
+import no.nav.su.se.bakover.web.søknad.ny.nyDigitalSøknad
+import no.nav.su.se.bakover.web.søknadsbehandling.beregning.beregn
+import no.nav.su.se.bakover.web.søknadsbehandling.bosituasjon.fullførBosituasjon
+import no.nav.su.se.bakover.web.søknadsbehandling.bosituasjon.taStillingTilEps
+import no.nav.su.se.bakover.web.søknadsbehandling.flyktning.leggTilFlyktningstatus
+import no.nav.su.se.bakover.web.søknadsbehandling.formue.leggTilFormue
+import no.nav.su.se.bakover.web.søknadsbehandling.iverksett.iverksett
+import no.nav.su.se.bakover.web.søknadsbehandling.ny.nySøknadsbehandling
+import no.nav.su.se.bakover.web.søknadsbehandling.opphold.leggTilFastOppholdINorge
+import no.nav.su.se.bakover.web.søknadsbehandling.opphold.leggTilInstitusjonsopphold
+import no.nav.su.se.bakover.web.søknadsbehandling.opphold.leggTilLovligOppholdINorge
+import no.nav.su.se.bakover.web.søknadsbehandling.opphold.leggTilUtenlandsopphold
+import no.nav.su.se.bakover.web.søknadsbehandling.oppmøte.leggTilPersonligOppmøte
+import no.nav.su.se.bakover.web.søknadsbehandling.sendTilAttestering.sendTilAttestering
+import no.nav.su.se.bakover.web.søknadsbehandling.simulering.simuler
+import no.nav.su.se.bakover.web.søknadsbehandling.uførhet.leggTilUføregrunnlag
+import no.nav.su.se.bakover.web.søknadsbehandling.virkningstidspunkt.leggTilVirkningstidspunkt
+import java.time.LocalDate
+
+/**
+ * Oppretter en ny sak med en ny digital søknad i den lokale postgres-instansen (bruker de samme endepunktene som frontend).
+ * Kan kjøres via `./local-db-scripts/create-søknadsbehandling.sh`
+ */
+fun main() {
```

Comment:
Hadde kanskje vært nyttig å kunne gjenbruke hele denne regla som en funksjon? F.eks for å kunne komponere mer "skreddersydde" tester (spesielt nyttig etterhvert som vi utvider med revurdering i tillegg)

Reply:
Trakk det ut i en felles funksjon og la til en test med assert :) Måtte inn med 4 json-ignore med TODO som vi egentlig ikke skulle ha vært der. Prøver fjerne i neste PR